datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/validators/validator.py

```diff
@@ -200,6 +200,42 @@ def extract_oid_from_symbol(self, metric, file):
             self.duplicated[OID] = self.used_oid[OID]
 
 
+class TableColumnHasTagValidator(ProfileValidator):
+    """
+    Validator responsible to check if all metrics related to a column in a profile
+    have tags to identify which row/entity they belong to
+    """
+
+    def validate(self, profile, path):
+        file_contents = find_profile_in_path(profile, path)
+        metrics = {}
+        if file_contents.get('metrics'):
+            metrics = file_contents.get('metrics')
+        for metric in metrics:
+            if metric.get('table'):
+                if not metric.get('metric_tags'):
+                    self.fail(
+                        "Table metric defined in line {} does not have tag associated".format(metric.get('__line__'))
```

Comment:
I think we should mentioned that `metric_tags` is missing. "not have tag associated" might not be clear enough for users.